src/annoylib.h

```diff
@@ -663,7 +815,7 @@ template<typename S, typename T, typename Distance, typename Random>
       // we have mmapped data
       close(_fd);
       off_t size = _n_nodes * _s;
-      munmap(_nodes, size);
+      munmap(_nodes, (size_t) size);
```

Comment:
why did you change this?

if anything should probably change `off_t size` to `size_t size` but i don't remember the difference between `off_t` and `size_t`

Reply:
might just be some win specific thing